src/cron/schedule-code-review.js

```diff
@@ -12,13 +12,18 @@ const {
 
 const STATUS_KEY = 'pr-request-review';
 
+const flatMappedStatusCommands = Object.entries(MAPPED_STATUS_COMMANDS).reduce(
+  (acc, [, statuses]) => ({ ...acc, ...statuses }),
+  {}
+);
+
```

Comment:
As you aren't using the keys of Object.entries, you could use Object.values instead, which only returns the value of the object without the key

```js
const flatMappedStatusCommands = Object.values(MAPPED_STATUS_COMMANDS).reduce(
  (acc, statuses) => ({ ...acc, ...statuses }),
  {}
);
```